### ISSUE TITLE
Model Overfits

### ISSUE THREAD

**Issue by HaysonC** [HaysonC/Encoder-DecoderLSTMTranslator#1]
Val loss stagnates; Not training.
